cypress/support/api/requests/Checkout.js

```diff
@@ -33,7 +34,7 @@ export function createCheckout({
   );
 
   const emailLine = getValueWithDefault(email, `email: "${email}"`);
-
+  cy.log("przed mutacją");
```

Comment:
Remove it.

Reply:
checkout shouldn't be changed under this task, changes removed